helm/provisioner/templates/coder.yaml

```diff
@@ -1,5 +1,7 @@
 ---
+{{- if not .Values.coder.serviceAccount.disableCreate }}
 {{ include "libcoder.serviceaccount" (list . "coder.serviceaccount") }}
+{{- end }}
```

Comment:
self-review: this was added for `helm/coder` in https://github.com/coder/coder/pull/14817 but not ported over here.

I'm not sure if it would be better to do it in libcoder, folks don't seem to look in there much.